uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -847,6 +847,76 @@ public function testFindOverlappingLeaveRequestsForOneOverlappingLeaveRequest()
     $this->assertEquals($leaveRequest1->id, $overlappingRequests[0]->id);
   }
 
+  public function testManagerCanCancelOrRejectUserLeaveRequestEvenIfBalanceChangeIsGreaterThanPeriodEntitlementBalanceChangeWhenAllowOveruseFalse() {
+    $manager = ContactFabricator::fabricate();
+    $staff = ContactFabricator::fabricate();
+    $entitlementFromDate = CRM_Utils_Date::processDate('2016-01-01');
+    $entitlementToDate = CRM_Utils_Date::processDate('2016-12-31');
+    $requestFromDate = new DateTime('2016-06-14');
+    $requestToDate = new DateTime('2016-06-14');
+    $requestFromDateType = $this->leaveRequestDayTypes['all_day']['value'];
+    $requestToDateType = $this->leaveRequestDayTypes['all_day']['value'];
```

Comment:
Just add `$dateType = $this->leaveRequestDayTypes['all_day']['value'];` and use the value for both from_date_type and to_date_type

Reply:
Done, but used requestDateType for better understanding.